[xtensor] build failure xsimd version mismatch

### Operating system

Windows

### Compiler

MSVC

### Steps to reproduce the behavior

```Shell
./vcpkg install xtensor[xsimd]:x64-windows
```


### Failure logs

[1/2] "C:/Users/aodhuble/Downloads/vcpkg/downloads/tools/cmake-3.27.1-windows/cmake-3.27.1-windows-i386/bin/cmake.exe" -E chdir "../../x64-windows-dbg" "C:/Users/aodhuble/Downloads/vcpkg/downloads/tools/cmake-3.27.1-windows/cmake-3.27.1-windows-i386/bin/cmake.exe" "C:/Users/aodhuble/Downloads/vcpkg/buildtrees/xtensor/src/0.24.6-35c4a680c8.clean" "-G" "Ninja" "-DCMAKE_BUILD_TYPE=Debug" "-DCMAKE_INSTALL_PREFIX=C:/Users/aodhuble/Downloads/vcpkg/packages/xtensor_x64-windows/debug" "-DFETCHCONTENT_FULLY_DISCONNECTED=ON" "-DXTENSOR_ENABLE_ASSERT=OFF" "-DXTENSOR_CHECK_DIMENSION=OFF" "-DBUILD_TESTS=OFF" "-DBUILD_BENCHMARK=OFF" "-DDOWNLOAD_GTEST=OFF" "-DDOWNLOAD_GBENCHMARK=OFF" "-DDEFAULT_COLUMN_MAJOR=OFF" "-DDISABLE_VS2017=OFF" "-DXTENSOR_USE_XSIMD=ON" "-DXTENSOR_USE_TBB=OFF" "-DCMAKE_MAKE_PROGRAM=C:/Program Files (x86)/Microsoft Visual Studio/2019/Enterprise/Common7/IDE/CommonExtensions/Microsoft/CMake/Ninja/ninja.exe" "-DBUILD_SHARED_LIBS=ON" "-DVCPKG_CHAINLOAD_TOOLCHAIN_FILE=C:/Users/aodhuble/Downloads/vcpkg/scripts/toolchains/windows.cmake" "-DVCPKG_TARGET_TRIPLET=x64-windows" "-DVCPKG_SET_CHARSET_FLAG=ON" "-DVCPKG_PLATFORM_TOOLSET=v142" "-DCMAKE_EXPORT_NO_PACKAGE_REGISTRY=ON" "-DCMAKE_FIND_PACKAGE_NO_PACKAGE_REGISTRY=ON" "-DCMAKE_FIND_PACKAGE_NO_SYSTEM_PACKAGE_REGISTRY=ON" "-DCMAKE_INSTALL_SYSTEM_RUNTIME_LIBS_SKIP=TRUE" "-DCMAKE_VERBOSE_MAKEFILE=ON" "-DVCPKG_APPLOCAL_DEPS=OFF" "-DCMAKE_TOOLCHAIN_FILE=C:/Users/aodhuble/Downloads/vcpkg/scripts/buildsystems/vcpkg.cmake" "-DCMAKE_ERROR_ON_ABSOLUTE_INSTALL_DESTINATION=ON" "-DVCPKG_CXX_FLAGS=" "-DVCPKG_CXX_FLAGS_RELEASE=" "-DVCPKG_CXX_FLAGS_DEBUG=" "-DVCPKG_C_FLAGS=" "-DVCPKG_C_FLAGS_RELEASE=" "-DVCPKG_C_FLAGS_DEBUG=" "-DVCPKG_CRT_LINKAGE=dynamic" "-DVCPKG_LINKER_FLAGS=" "-DVCPKG_LINKER_FLAGS_RELEASE=" "-DVCPKG_LINKER_FLAGS_DEBUG=" "-DVCPKG_TARGET_ARCHITECTURE=x64" "-DCMAKE_INSTALL_LIBDIR:STRING=lib" "-DCMAKE_INSTALL_BINDIR:STRING=bin" "-D_VCPKG_ROOT_DIR=C:/Users/aodhuble/Downloads/vcpkg" "-D_VCPKG_INSTALLED_DIR=C:/Users/aodhuble/Downloads/vcpkg/installed" "-DVCPKG_MANIFEST_INSTALL=OFF"
FAILED: ../../x64-windows-dbg/CMakeCache.txt 
"C:/Users/aodhuble/Downloads/vcpkg/downloads/tools/cmake-3.27.1-windows/cmake-3.27.1-windows-i386/bin/cmake.exe" -E chdir "../../x64-windows-dbg" "C:/Users/aodhuble/Downloads/vcpkg/downloads/tools/cmake-3.27.1-windows/cmake-3.27.1-windows-i386/bin/cmake.exe" "C:/Users/aodhuble/Downloads/vcpkg/buildtrees/xtensor/src/0.24.6-35c4a680c8.clean" "-G" "Ninja" "-DCMAKE_BUILD_TYPE=Debug" "-DCMAKE_INSTALL_PREFIX=C:/Users/aodhuble/Downloads/vcpkg/packages/xtensor_x64-windows/debug" "-DFETCHCONTENT_FULLY_DISCONNECTED=ON" "-DXTENSOR_ENABLE_ASSERT=OFF" "-DXTENSOR_CHECK_DIMENSION=OFF" "-DBUILD_TESTS=OFF" "-DBUILD_BENCHMARK=OFF" "-DDOWNLOAD_GTEST=OFF" "-DDOWNLOAD_GBENCHMARK=OFF" "-DDEFAULT_COLUMN_MAJOR=OFF" "-DDISABLE_VS2017=OFF" "-DXTENSOR_USE_XSIMD=ON" "-DXTENSOR_USE_TBB=OFF" "-DCMAKE_MAKE_PROGRAM=C:/Program Files (x86)/Microsoft Visual Studio/2019/Enterprise/Common7/IDE/CommonExtensions/Microsoft/CMake/Ninja/ninja.exe" "-DBUILD_SHARED_LIBS=ON" "-DVCPKG_CHAINLOAD_TOOLCHAIN_FILE=C:/Users/aodhuble/Downloads/vcpkg/scripts/toolchains/windows.cmake" "-DVCPKG_TARGET_TRIPLET=x64-windows" "-DVCPKG_SET_CHARSET_FLAG=ON" "-DVCPKG_PLATFORM_TOOLSET=v142" "-DCMAKE_EXPORT_NO_PACKAGE_REGISTRY=ON" "-DCMAKE_FIND_PACKAGE_NO_PACKAGE_REGISTRY=ON" "-DCMAKE_FIND_PACKAGE_NO_SYSTEM_PACKAGE_REGISTRY=ON" "-DCMAKE_INSTALL_SYSTEM_RUNTIME_LIBS_SKIP=TRUE" "-DCMAKE_VERBOSE_MAKEFILE=ON" "-DVCPKG_APPLOCAL_DEPS=OFF" "-DCMAKE_TOOLCHAIN_FILE=C:/Users/aodhuble/Downloads/vcpkg/scripts/buildsystems/vcpkg.cmake" "-DCMAKE_ERROR_ON_ABSOLUTE_INSTALL_DESTINATION=ON" "-DVCPKG_CXX_FLAGS=" "-DVCPKG_CXX_FLAGS_RELEASE=" "-DVCPKG_CXX_FLAGS_DEBUG=" "-DVCPKG_C_FLAGS=" "-DVCPKG_C_FLAGS_RELEASE=" "-DVCPKG_C_FLAGS_DEBUG=" "-DVCPKG_CRT_LINKAGE=dynamic" "-DVCPKG_LINKER_FLAGS=" "-DVCPKG_LINKER_FLAGS_RELEASE=" "-DVCPKG_LINKER_FLAGS_DEBUG=" "-DVCPKG_TARGET_ARCHITECTURE=x64" "-DCMAKE_INSTALL_LIBDIR:STRING=lib" "-DCMAKE_INSTALL_BINDIR:STRING=bin" "-D_VCPKG_ROOT_DIR=C:/Users/aodhuble/Downloads/vcpkg" "-D_VCPKG_INSTALLED_DIR=C:/Users/aodhuble/Downloads/vcpkg/installed" "-DVCPKG_MANIFEST_INSTALL=OFF"
CMake Deprecation Warning at CMakeLists.txt:10 (cmake_minimum_required):
  Compatibility with CMake < 3.5 will be removed from a future version of
  CMake.

  Update the VERSION argument <min> value or use a ...<max> suffix to tell
  CMake that the project does not need compatibility with older versions.


-- The CXX compiler identification is MSVC 19.29.30151.0
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Check for working CXX compiler: C:/Program Files (x86)/Microsoft Visual Studio/2019/Enterprise/VC/Tools/MSVC/14.29.30133/bin/Hostx64/x64/cl.exe - skipped
-- Detecting CXX compile features
-- Detecting CXX compile features - done
-- Building xtensor v0.24.6
-- Found xtl: C:/Users/aodhuble/Downloads/vcpkg/installed/x64-windows/include/xtl
CMake Error at C:/Users/aodhuble/Downloads/vcpkg/scripts/buildsystems/vcpkg.cmake:855 (_find_package):
  Could not find a configuration file for package "xsimd" that is compatible
  with requested version "10.0.0".

  The following configuration files were considered but not accepted:

    C:/Users/aodhuble/Downloads/vcpkg/installed/x64-windows/share/xsimd/xsimdConfig.cmake, version: 11.0.1

Call Stack (most recent call first):
  CMakeLists.txt:72 (find_package)


-- Configuring incomplete, errors occurred!
[2/2] "C:/Users/aodhuble/Downloads/vcpkg/downloads/tools/cmake-3.27.1-windows/cmake-3.27.1-windows-i386/bin/cmake.exe" -E chdir ".." "C:/Users/aodhuble/Downloads/vcpkg/downloads/tools/cmake-3.27.1-windows/cmake-3.27.1-windows-i386/bin/cmake.exe" "C:/Users/aodhuble/Downloads/vcpkg/buildtrees/xtensor/src/0.24.6-35c4a680c8.clean" "-G" "Ninja" "-DCMAKE_BUILD_TYPE=Release" "-DCMAKE_INSTALL_PREFIX=C:/Users/aodhuble/Downloads/vcpkg/packages/xtensor_x64-windows" "-DFETCHCONTENT_FULLY_DISCONNECTED=ON" "-DXTENSOR_ENABLE_ASSERT=OFF" "-DXTENSOR_CHECK_DIMENSION=OFF" "-DBUILD_TESTS=OFF" "-DBUILD_BENCHMARK=OFF" "-DDOWNLOAD_GTEST=OFF" "-DDOWNLOAD_GBENCHMARK=OFF" "-DDEFAULT_COLUMN_MAJOR=OFF" "-DDISABLE_VS2017=OFF" "-DXTENSOR_USE_XSIMD=ON" "-DXTENSOR_USE_TBB=OFF" "-DCMAKE_MAKE_PROGRAM=C:/Program Files (x86)/Microsoft Visual Studio/2019/Enterprise/Common7/IDE/CommonExtensions/Microsoft/CMake/Ninja/ninja.exe" "-DBUILD_SHARED_LIBS=ON" "-DVCPKG_CHAINLOAD_TOOLCHAIN_FILE=C:/Users/aodhuble/Downloads/vcpkg/scripts/toolchains/windows.cmake" "-DVCPKG_TARGET_TRIPLET=x64-windows" "-DVCPKG_SET_CHARSET_FLAG=ON" "-DVCPKG_PLATFORM_TOOLSET=v142" "-DCMAKE_EXPORT_NO_PACKAGE_REGISTRY=ON" "-DCMAKE_FIND_PACKAGE_NO_PACKAGE_REGISTRY=ON" "-DCMAKE_FIND_PACKAGE_NO_SYSTEM_PACKAGE_REGISTRY=ON" "-DCMAKE_INSTALL_SYSTEM_RUNTIME_LIBS_SKIP=TRUE" "-DCMAKE_VERBOSE_MAKEFILE=ON" "-DVCPKG_APPLOCAL_DEPS=OFF" "-DCMAKE_TOOLCHAIN_FILE=C:/Users/aodhuble/Downloads/vcpkg/scripts/buildsystems/vcpkg.cmake" "-DCMAKE_ERROR_ON_ABSOLUTE_INSTALL_DESTINATION=ON" "-DVCPKG_CXX_FLAGS=" "-DVCPKG_CXX_FLAGS_RELEASE=" "-DVCPKG_CXX_FLAGS_DEBUG=" "-DVCPKG_C_FLAGS=" "-DVCPKG_C_FLAGS_RELEASE=" "-DVCPKG_C_FLAGS_DEBUG=" "-DVCPKG_CRT_LINKAGE=dynamic" "-DVCPKG_LINKER_FLAGS=" "-DVCPKG_LINKER_FLAGS_RELEASE=" "-DVCPKG_LINKER_FLAGS_DEBUG=" "-DVCPKG_TARGET_ARCHITECTURE=x64" "-DCMAKE_INSTALL_LIBDIR:STRING=lib" "-DCMAKE_INSTALL_BINDIR:STRING=bin" "-D_VCPKG_ROOT_DIR=C:/Users/aodhuble/Downloads/vcpkg" "-D_VCPKG_INSTALLED_DIR=C:/Users/aodhuble/Downloads/vcpkg/installed" "-DVCPKG_MANIFEST_INSTALL=OFF"
FAILED: ../CMakeCache.txt 
"C:/Users/aodhuble/Downloads/vcpkg/downloads/tools/cmake-3.27.1-windows/cmake-3.27.1-windows-i386/bin/cmake.exe" -E chdir ".." "C:/Users/aodhuble/Downloads/vcpkg/downloads/tools/cmake-3.27.1-windows/cmake-3.27.1-windows-i386/bin/cmake.exe" "C:/Users/aodhuble/Downloads/vcpkg/buildtrees/xtensor/src/0.24.6-35c4a680c8.clean" "-G" "Ninja" "-DCMAKE_BUILD_TYPE=Release" "-DCMAKE_INSTALL_PREFIX=C:/Users/aodhuble/Downloads/vcpkg/packages/xtensor_x64-windows" "-DFETCHCONTENT_FULLY_DISCONNECTED=ON" "-DXTENSOR_ENABLE_ASSERT=OFF" "-DXTENSOR_CHECK_DIMENSION=OFF" "-DBUILD_TESTS=OFF" "-DBUILD_BENCHMARK=OFF" "-DDOWNLOAD_GTEST=OFF" "-DDOWNLOAD_GBENCHMARK=OFF" "-DDEFAULT_COLUMN_MAJOR=OFF" "-DDISABLE_VS2017=OFF" "-DXTENSOR_USE_XSIMD=ON" "-DXTENSOR_USE_TBB=OFF" "-DCMAKE_MAKE_PROGRAM=C:/Program Files (x86)/Microsoft Visual Studio/2019/Enterprise/Common7/IDE/CommonExtensions/Microsoft/CMake/Ninja/ninja.exe" "-DBUILD_SHARED_LIBS=ON" "-DVCPKG_CHAINLOAD_TOOLCHAIN_FILE=C:/Users/aodhuble/Downloads/vcpkg/scripts/toolchains/windows.cmake" "-DVCPKG_TARGET_TRIPLET=x64-windows" "-DVCPKG_SET_CHARSET_FLAG=ON" "-DVCPKG_PLATFORM_TOOLSET=v142" "-DCMAKE_EXPORT_NO_PACKAGE_REGISTRY=ON" "-DCMAKE_FIND_PACKAGE_NO_PACKAGE_REGISTRY=ON" "-DCMAKE_FIND_PACKAGE_NO_SYSTEM_PACKAGE_REGISTRY=ON" "-DCMAKE_INSTALL_SYSTEM_RUNTIME_LIBS_SKIP=TRUE" "-DCMAKE_VERBOSE_MAKEFILE=ON" "-DVCPKG_APPLOCAL_DEPS=OFF" "-DCMAKE_TOOLCHAIN_FILE=C:/Users/aodhuble/Downloads/vcpkg/scripts/buildsystems/vcpkg.cmake" "-DCMAKE_ERROR_ON_ABSOLUTE_INSTALL_DESTINATION=ON" "-DVCPKG_CXX_FLAGS=" "-DVCPKG_CXX_FLAGS_RELEASE=" "-DVCPKG_CXX_FLAGS_DEBUG=" "-DVCPKG_C_FLAGS=" "-DVCPKG_C_FLAGS_RELEASE=" "-DVCPKG_C_FLAGS_DEBUG=" "-DVCPKG_CRT_LINKAGE=dynamic" "-DVCPKG_LINKER_FLAGS=" "-DVCPKG_LINKER_FLAGS_RELEASE=" "-DVCPKG_LINKER_FLAGS_DEBUG=" "-DVCPKG_TARGET_ARCHITECTURE=x64" "-DCMAKE_INSTALL_LIBDIR:STRING=lib" "-DCMAKE_INSTALL_BINDIR:STRING=bin" "-D_VCPKG_ROOT_DIR=C:/Users/aodhuble/Downloads/vcpkg" "-D_VCPKG_INSTALLED_DIR=C:/Users/aodhuble/Downloads/vcpkg/installed" "-DVCPKG_MANIFEST_INSTALL=OFF"
CMake Deprecation Warning at CMakeLists.txt:10 (cmake_minimum_required):
  Compatibility with CMake < 3.5 will be removed from a future version of
  CMake.

  Update the VERSION argument <min> value or use a ...<max> suffix to tell
  CMake that the project does not need compatibility with older versions.


-- The CXX compiler identification is MSVC 19.29.30151.0
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Check for working CXX compiler: C:/Program Files (x86)/Microsoft Visual Studio/2019/Enterprise/VC/Tools/MSVC/14.29.30133/bin/Hostx64/x64/cl.exe - skipped
-- Detecting CXX compile features
-- Detecting CXX compile features - done
-- Building xtensor v0.24.6
-- Found xtl: C:/Users/aodhuble/Downloads/vcpkg/installed/x64-windows/include/xtl
CMake Error at C:/Users/aodhuble/Downloads/vcpkg/scripts/buildsystems/vcpkg.cmake:855 (_find_package):
  Could not find a configuration file for package "xsimd" that is compatible
  with requested version "10.0.0".

  The following configuration files were considered but not accepted:

    C:/Users/aodhuble/Downloads/vcpkg/installed/x64-windows/share/xsimd/xsimdConfig.cmake, version: 11.0.1

Call Stack (most recent call first):
  CMakeLists.txt:72 (find_package)


-- Configuring incomplete, errors occurred!
ninja: build stopped: subcommand failed.


### Additional context

Xsimd and xtensor versions are out of step

--- a/ports/xtensor/fix-find-xsimd.patch
+++ b/ports/xtensor/fix-find-xsimd.patch
@@ -1,13 +1,31 @@
 diff --git a/CMakeLists.txt b/CMakeLists.txt
-index 5758320..522a94c 100644
+index 5c93655..f02dc48 100644
 --- a/CMakeLists.txt
 +++ b/CMakeLists.txt
-@@ -59,7 +59,7 @@ if(XTENSOR_USE_TBB AND XTENSOR_USE_OPENMP)
+@@ -69,7 +69,12 @@ if(XTENSOR_USE_XSIMD)
+             message(STATUS "Found xsimd v${xsimd_VERSION}")
+         endif()
+     else()
+-        find_package(xsimd ${xsimd_REQUIRED_VERSION} REQUIRED)
++        find_package(xsimd CONFIG REQUIRED)
++        if(${xsimd_VERSION} GREATER_EQUAL ${xsimd_REQUIRED_VERSION})
++            set(xsimd_REQUIRED_VERSION ${xsimd_VERSION})
++        else()
++            message(ERROR "Mismatch xsimd versions. Found '${xsimd_VERSION}' but requires: '${xsimd_REQUIRED_VERSION}'")
++        endif()
+         message(STATUS "Found xsimd: ${xsimd_INCLUDE_DIRS}/xsimd")
+     endif()
+ endif()
+diff --git a/xtensorConfig.cmake.in b/xtensorConfig.cmake.in
+index ec72abb..7825612 100644
+--- a/xtensorConfig.cmake.in
++++ b/xtensorConfig.cmake.in
+@@ -25,7 +25,7 @@ if(NOT TARGET @PROJECT_NAME@)
  endif()
  
  if(XTENSOR_USE_XSIMD)
--    set(xsimd_REQUIRED_VERSION 10.0.0)
-+    set(xsimd_REQUIRED_VERSION 11.0.1)
-     if(TARGET xsimd)
-         set(xsimd_VERSION ${XSIMD_VERSION_MAJOR}.${XSIMD_VERSION_MINOR}.${XSIMD_VERSION_PATCH})
-         # Note: This is not SEMVER compatible comparison
+-    find_dependency(xsimd @xsimd_REQUIRED_VERSION@)
++    find_dependency(xsimd @xsimd_REQUIRED_VERSION@ CONFIG)
+     target_link_libraries(@PROJECT_NAME@ INTERFACE xsimd)
+     target_compile_definitions(@PROJECT_NAME@ INTERFACE XTENSOR_USE_XSIMD)
+ endif()

--- a/ports/xtensor/fix-find-xsimd.patch
+++ b/ports/xtensor/fix-find-xsimd.patch
@@ -1,0 +1,13 @@
+diff --git a/CMakeLists.txt b/CMakeLists.txt
+index 5758320..522a94c 100644
+--- a/CMakeLists.txt
++++ b/CMakeLists.txt
+@@ -59,7 +59,7 @@ if(XTENSOR_USE_TBB AND XTENSOR_USE_OPENMP)
+ endif()
+ 
+ if(XTENSOR_USE_XSIMD)
+-    set(xsimd_REQUIRED_VERSION 10.0.0)
++    set(xsimd_REQUIRED_VERSION 11.0.1)
+     if(TARGET xsimd)
+         set(xsimd_VERSION ${XSIMD_VERSION_MAJOR}.${XSIMD_VERSION_MINOR}.${XSIMD_VERSION_PATCH})
+         # Note: This is not SEMVER compatible comparison

--- a/ports/xtensor/portfile.cmake
+++ b/ports/xtensor/portfile.cmake
@@ -8,6 +8,7 @@ vcpkg_from_github(
     HEAD_REF master
     PATCHES
         fix-find-tbb-and-install-destination.patch
+        fix-find-xsimd.patch
 )
 
 vcpkg_check_features(OUT_FEATURE_OPTIONS FEATURE_OPTIONS

--- a/ports/xtensor/vcpkg.json
+++ b/ports/xtensor/vcpkg.json
@@ -1,6 +1,7 @@
 {
   "name": "xtensor",
   "version": "0.24.6",
+  "port-version": 1,
   "description": "C++ tensors with broadcasting and lazy computing",
   "homepage": "https://github.com/xtensor-stack/xtensor",
   "license": "BSD-3-Clause",

--- a/versions/baseline.json
+++ b/versions/baseline.json
@@ -8922,7 +8922,7 @@
     },
     "xtensor": {
       "baseline": "0.24.6",
-      "port-version": 0
+      "port-version": 1
     },
     "xtensor-blas": {
       "baseline": "0.20.0",

--- a/versions/x-/xtensor.json
+++ b/versions/x-/xtensor.json
@@ -1,6 +1,11 @@
 {
   "versions": [
     {
+      "git-tree": "3f45c9f7970a7a556d00e78ba35764b2f85bb50c",
+      "version": "0.24.6",
+      "port-version": 1
+    },
+    {
       "git-tree": "aeb0efc2bdf326191f5dff18bb2506690902b162",
       "version": "0.24.6",
       "port-version": 0

--- a/versions/x-/xtensor.json
+++ b/versions/x-/xtensor.json
@@ -1,7 +1,7 @@
 {
   "versions": [
     {
-      "git-tree": "3f45c9f7970a7a556d00e78ba35764b2f85bb50c",
+      "git-tree": "3243d2557778b77adbd7b2dbc83ade2456e8fe5c",
       "version": "0.24.6",
       "port-version": 1
     },

